README.md

```diff
@@ -75,9 +76,26 @@ scrape_configs:
 
 `honor_labels: true` is important because exporter returns metrics with `instance` label set.
 
+### Docker
+#### Building
+Clone repo and build it as usual docker image:
+```
+dokcer build . -t rds_exporter 
+```
+
+#### Launching
+* Pass credentials either via `AWS_ACCESS_KEY_ID`/ `AWS_SECRET_ACCESS_KEY` environment variables or via `~/.aws/credentials` file
+* Expose port for exporter
+* Pass `USE_FILTER` environment variable if you want to filter instances by tag  
+
+```
+docker run -p 9042:9042 -e USE_FILTER=true -e AWS_ACCESS_KEY_ID=<xxxxx> -e AWS_SECRET_ACCESS_KEY=<yyyyy> rds-exporter
+```
+
+
 ## Metrics
 
-Exporter synthesizes [node_exporter](https://github.com/prometheus/node_exporter)-like metrics where possible.
+Exporter synthesizes [node_exporter](https://github.com/prometheus/node_exporter) -like metrics where possible.
```

Comment:
this sentence is not clear

Reply:
This sentence is from initial Readme file.
In my opinion it means that RDS exporter synthesizes hardware and OS metrics as well as node exporter

Reply:
then write your sentence. is definitely more clear than the one above

Reply:
Fixed in https://github.com/pdffillerdocker/rds_exporter/pull/1/commits/d3c0474d59a3a04b4e1fb512139aaced0ac831ff